source/slang-capture-replay/output-stream.cpp

```diff
@@ -26,7 +26,7 @@ namespace SlangCapture
 
     void FileOutputStream::write(const void* data, size_t len)
     {
-        SLANG_CAPTURE_ASSERT(m_fileStream.write(data, len));
+        SLANG_CAPTURE_ASSERT(SLANG_OK == m_fileStream.write(data, len));
```

Comment:
I agree to rename the macro for a different reason.
When a macro has a name with "ASSERT", it means the evaluation wrapped with the macro will be removed in release build.
SLANG_RELEASE_ASSERT is an exception and it explicitly has "RELEASE" keyword in it.
For cases that works like assert but has to be executed in all build configuration, "SANITY", "CHECK", or "VERIFY" are often used in order to avoid keyword  "ASSERT".

Reply:
OK, will change.
This is some old habit that I learnt when working with the autosar.